++ ls caseresult/ -ltr
++ wc -l
+ filename=case5
+ mkdir caseresult/case5
+ RESULT_DIR=/home/hadoop/git/physical_design/caseresult/case5
+ TEN=10
+ SLEEP_TIME=300
+ YCSB_DIR=/home/hadoop/ycsb-0.1.4
+ number_of_operations=5000000
+ number_of_threads=20
+ number_of_records=5000000
+ output_file_suffix=.dat
+ experiment_list_file=/home/hadoop/git/physical_design/case5.list
+ clean=false
+ create=false
+ load=false
+ random_read=false
+ scan=false
+ all=false
+ for i in '$@'
+ '[' all == all ']'
+ all=true
+ let counter=1
+ cd /home/hadoop/ycsb-0.1.4
++ pwd
+ echo /home/hadoop/ycsb-0.1.4
/home/hadoop/ycsb-0.1.4
+ read line
+ true
+ echo 'droping usertable...'
droping usertable...
+ hbase shell /home/hadoop/git/physical_design/delete_table
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/hadoop/hbase-0.98.3-hadoop1/lib/slf4j-log4j12-1.6.4.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/hadoop/hadoop-1.2.1/lib/slf4j-log4j12-1.4.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
0 row(s) in 34.9940 seconds

0 row(s) in 0.7990 seconds

0 row(s) in 0.0680 seconds

+ echo 'sleeping for 300 ...'
sleeping for 300 ...
+ sleep 300
+ true
+ echo 'creating usertable...'
creating usertable...
+ create_statement='create '\''usertable'\'', { NAME=>'\''family'\'', DATA_BLOCK_ENCODING=>'\''PREFIX_TREE'\''}, SPLITS => ['\''user1'\'', '\''user2'\'', '\''user3'\'', '\''user4'\'', '\''user5'\'', '\''user6'\'', '\''user7'\'', '\''user8'\'', '\''user9'\'']'
+ echo create ''\''usertable'\'',' '{' 'NAME=>'\''family'\'',' 'DATA_BLOCK_ENCODING=>'\''PREFIX_TREE'\''},' SPLITS '=>' '['\''user1'\'',' ''\''user2'\'',' ''\''user3'\'',' ''\''user4'\'',' ''\''user5'\'',' ''\''user6'\'',' ''\''user7'\'',' ''\''user8'\'',' ''\''user9'\'']'
+ echo 'exit '
+ create_table /home/hadoop/git/physical_design/caseresult/case5/create_1.script
+ echo 'creating table...'
creating table...
+ echo /home/hadoop/git/physical_design/caseresult/case5/create_1.script
/home/hadoop/git/physical_design/caseresult/case5/create_1.script
+ hbase shell /home/hadoop/git/physical_design/caseresult/case5/create_1.script
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/hadoop/hbase-0.98.3-hadoop1/lib/slf4j-log4j12-1.6.4.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/hadoop/hadoop-1.2.1/lib/slf4j-log4j12-1.4.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
0 row(s) in 1.6150 seconds

+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo 'loading data...'
loading data...
+ echo 'starting time: '
starting time: 
++ date +%s
+ echo 1404345980
1404345980
++ echo /home/hadoop/git/physical_design/caseresult/case5/load_1.dat
+ ./ld 5000000 20 /home/hadoop/git/physical_design/caseresult/case5/load_1.dat
Loading workload...
Starting test.
2014-07-02 17:06:21:582 0 sec: 0 operations; 
14/07/02 17:06:22 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/02 17:06:22 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/02 17:06:22 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/02 17:06:22 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/02 17:06:22 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/02 17:06:22 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/02 17:06:22 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/02 17:06:22 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/02 17:06:22 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/02 17:06:22 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/02 17:06:22 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/02 17:06:22 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/02 17:06:22 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/02 17:06:22 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/02 17:06:22 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/02 17:06:22 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0x41fbfa37, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/02 17:06:22 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0x41fbfa37 connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/02 17:06:22 INFO zookeeper.ClientCnxn: Opening socket connection to server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/02 17:06:22 INFO zookeeper.ClientCnxn: Socket connection established to sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, initiating session
14/07/02 17:06:22 INFO zookeeper.ClientCnxn: Session establishment complete on server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, sessionid = 0x46f985a17c000d, negotiated timeout = 90000
2014-07-02 17:06:31:583 10 sec: 242974 operations; 24287.68 current ops/sec; [INSERT AverageLatency(us)=514.65] 
2014-07-02 17:06:41:585 20 sec: 242974 operations; 0 current ops/sec;  
2014-07-02 17:06:51:586 30 sec: 242974 operations; 0 current ops/sec;  
2014-07-02 17:07:01:829 40 sec: 576626 operations; 32570.48 current ops/sec; [INSERT AverageLatency(us)=759.34] 
2014-07-02 17:07:11:830 50 sec: 664018 operations; 8738.33 current ops/sec; [INSERT AverageLatency(us)=199.33] 
2014-07-02 17:07:21:831 60 sec: 664018 operations; 0 current ops/sec;  
2014-07-02 17:07:31:832 70 sec: 664172 operations; 15.4 current ops/sec; [INSERT AverageLatency(us)=357435.34] 
14/07/02 17:07:32 INFO client.AsyncProcess: #20, table=usertable, attempt=10/35 failed 2458 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10039 ms, replay 2458 ops.
14/07/02 17:07:32 INFO client.AsyncProcess: #1, table=usertable, attempt=10/35 failed 2382 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10027 ms, replay 2382 ops.
14/07/02 17:07:32 INFO client.AsyncProcess: #10, table=usertable, attempt=10/35 failed 2511 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10029 ms, replay 2511 ops.
14/07/02 17:07:32 INFO client.AsyncProcess: #12, table=usertable, attempt=10/35 failed 2383 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10060 ms, replay 2383 ops.
14/07/02 17:07:32 INFO client.AsyncProcess: #13, table=usertable, attempt=10/35 failed 2416 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10067 ms, replay 2416 ops.
14/07/02 17:07:32 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 2495 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10055 ms, replay 2495 ops.
14/07/02 17:07:32 INFO client.AsyncProcess: #6, table=usertable, attempt=10/35 failed 2496 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10080 ms, replay 2496 ops.
14/07/02 17:07:32 INFO client.AsyncProcess: #2, table=usertable, attempt=10/35 failed 2443 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10064 ms, replay 2443 ops.
14/07/02 17:07:32 INFO client.AsyncProcess: #19, table=usertable, attempt=10/35 failed 2404 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10035 ms, replay 2404 ops.
14/07/02 17:07:32 INFO client.AsyncProcess: #4, table=usertable, attempt=10/35 failed 2403 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10048 ms, replay 2403 ops.
14/07/02 17:07:32 INFO client.AsyncProcess: #17, table=usertable, attempt=10/35 failed 2477 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10038 ms, replay 2477 ops.
14/07/02 17:07:32 INFO client.AsyncProcess: #5, table=usertable, attempt=10/35 failed 2390 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10098 ms, replay 2390 ops.
2014-07-02 17:07:41:833 80 sec: 736669 operations; 7248.98 current ops/sec; [INSERT AverageLatency(us)=3422.46] 
14/07/02 17:07:44 INFO client.AsyncProcess: #10, table=usertable, attempt=11/35 failed 2511 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=920, waitTime=2652, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10014 ms, replay 2511 ops.
14/07/02 17:07:44 INFO client.AsyncProcess: #12, table=usertable, attempt=11/35 failed 2383 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=920, waitTime=2652, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10059 ms, replay 2383 ops.
14/07/02 17:07:44 INFO client.AsyncProcess: #13, table=usertable, attempt=11/35 failed 2416 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=920, waitTime=2652, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10054 ms, replay 2416 ops.
14/07/02 17:07:44 INFO client.AsyncProcess: #1, table=usertable, attempt=11/35 failed 2382 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=920, waitTime=2652, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10011 ms, replay 2382 ops.
14/07/02 17:07:44 INFO client.AsyncProcess: #5, table=usertable, attempt=11/35 failed 2390 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=920, waitTime=2652, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10001 ms, replay 2390 ops.
14/07/02 17:07:44 INFO client.AsyncProcess: #19, table=usertable, attempt=11/35 failed 2404 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=920, waitTime=2652, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10078 ms, replay 2404 ops.
14/07/02 17:07:44 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 failed 2495 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=920, waitTime=2652, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10029 ms, replay 2495 ops.
14/07/02 17:07:44 INFO client.AsyncProcess: #20, table=usertable, attempt=11/35 failed 2458 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=920, waitTime=2652, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10024 ms, replay 2458 ops.
14/07/02 17:07:44 INFO client.AsyncProcess: #6, table=usertable, attempt=11/35 failed 2496 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=920, waitTime=2652, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10096 ms, replay 2496 ops.
14/07/02 17:07:44 INFO client.AsyncProcess: #2, table=usertable, attempt=11/35 failed 2443 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=920, waitTime=2652, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10094 ms, replay 2443 ops.
14/07/02 17:07:44 INFO client.AsyncProcess: #4, table=usertable, attempt=11/35 failed 2403 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=920, waitTime=2652, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10028 ms, replay 2403 ops.
14/07/02 17:07:44 INFO client.AsyncProcess: #17, table=usertable, attempt=11/35 failed 2477 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=920, waitTime=2652, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 10031 ms, replay 2477 ops.
2014-07-02 17:07:51:834 90 sec: 834848 operations; 9816.92 current ops/sec; [INSERT AverageLatency(us)=306.75] 
14/07/02 17:07:58 INFO client.AsyncProcess: #1, table=usertable, attempt=12/35 failed 2382 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20163 ms, replay 2382 ops.
14/07/02 17:07:58 INFO client.AsyncProcess: #6, table=usertable, attempt=12/35 failed 2496 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20125 ms, replay 2496 ops.
14/07/02 17:07:58 INFO client.AsyncProcess: #12, table=usertable, attempt=12/35 failed 2383 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20165 ms, replay 2383 ops.
14/07/02 17:07:58 INFO client.AsyncProcess: #10, table=usertable, attempt=12/35 failed 2511 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20045 ms, replay 2511 ops.
14/07/02 17:07:58 INFO client.AsyncProcess: #5, table=usertable, attempt=12/35 failed 2390 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20007 ms, replay 2390 ops.
14/07/02 17:07:58 INFO client.AsyncProcess: #18, table=usertable, attempt=12/35 failed 2495 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20015 ms, replay 2495 ops.
14/07/02 17:07:58 INFO client.AsyncProcess: #13, table=usertable, attempt=12/35 failed 2416 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20127 ms, replay 2416 ops.
14/07/02 17:07:58 INFO client.AsyncProcess: #17, table=usertable, attempt=12/35 failed 2477 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20056 ms, replay 2477 ops.
14/07/02 17:07:58 INFO client.AsyncProcess: #20, table=usertable, attempt=12/35 failed 2458 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20125 ms, replay 2458 ops.
14/07/02 17:07:58 INFO client.AsyncProcess: #19, table=usertable, attempt=12/35 failed 2404 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20056 ms, replay 2404 ops.
14/07/02 17:07:58 INFO client.AsyncProcess: #4, table=usertable, attempt=12/35 failed 2403 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20065 ms, replay 2403 ops.
14/07/02 17:07:58 INFO client.AsyncProcess: #2, table=usertable, attempt=12/35 failed 2443 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:42242 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20128 ms, replay 2443 ops.
2014-07-02 17:08:01:835 100 sec: 888229 operations; 5338.1 current ops/sec; [INSERT AverageLatency(us)=1427.4] 
2014-07-02 17:08:11:835 110 sec: 1026079 operations; 13783.62 current ops/sec; [INSERT AverageLatency(us)=785.59] 
14/07/02 17:08:20 INFO client.AsyncProcess: #20, table=usertable, attempt=13/35 failed 2458 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1509, waitTime=3726, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20197 ms, replay 2458 ops.
14/07/02 17:08:20 INFO client.AsyncProcess: #13, table=usertable, attempt=13/35 failed 2416 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1509, waitTime=3726, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20041 ms, replay 2416 ops.
14/07/02 17:08:20 INFO client.AsyncProcess: #10, table=usertable, attempt=13/35 failed 2511 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1509, waitTime=3726, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20082 ms, replay 2511 ops.
14/07/02 17:08:20 INFO client.AsyncProcess: #12, table=usertable, attempt=13/35 failed 2383 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1509, waitTime=3726, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20140 ms, replay 2383 ops.
14/07/02 17:08:20 INFO client.AsyncProcess: #17, table=usertable, attempt=13/35 failed 2477 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1509, waitTime=3726, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20139 ms, replay 2477 ops.
14/07/02 17:08:20 INFO client.AsyncProcess: #18, table=usertable, attempt=13/35 failed 2495 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1509, waitTime=3726, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20189 ms, replay 2495 ops.
14/07/02 17:08:20 INFO client.AsyncProcess: #1, table=usertable, attempt=13/35 failed 2382 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1509, waitTime=3726, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20112 ms, replay 2382 ops.
14/07/02 17:08:20 INFO client.AsyncProcess: #5, table=usertable, attempt=13/35 failed 2390 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1509, waitTime=3726, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20168 ms, replay 2390 ops.
14/07/02 17:08:20 INFO client.AsyncProcess: #19, table=usertable, attempt=13/35 failed 2404 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1509, waitTime=3726, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20014 ms, replay 2404 ops.
14/07/02 17:08:20 INFO client.AsyncProcess: #4, table=usertable, attempt=13/35 failed 2403 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1509, waitTime=3726, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20161 ms, replay 2403 ops.
14/07/02 17:08:20 INFO client.AsyncProcess: #6, table=usertable, attempt=13/35 failed 2496 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1509, waitTime=3726, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20145 ms, replay 2496 ops.
14/07/02 17:08:20 INFO client.AsyncProcess: #2, table=usertable, attempt=13/35 failed 2443 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1509, waitTime=3726, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20174 ms, replay 2443 ops.
2014-07-02 17:08:21:836 120 sec: 1212490 operations; 18639.24 current ops/sec; [INSERT AverageLatency(us)=367.98] 
2014-07-02 17:08:31:836 130 sec: 1212490 operations; 0 current ops/sec;  
2014-07-02 17:08:41:837 140 sec: 1409753 operations; 19724.33 current ops/sec; [INSERT AverageLatency(us)=981.21] 
14/07/02 17:08:43 INFO client.AsyncProcess: #13, table=usertable, attempt=14/35 failed 2416 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1837, waitTime=2220, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20148 ms, replay 2416 ops.
14/07/02 17:08:43 INFO client.AsyncProcess: #5, table=usertable, attempt=14/35 failed 2390 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1837, waitTime=2220, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20045 ms, replay 2390 ops.
14/07/02 17:08:43 INFO client.AsyncProcess: #1, table=usertable, attempt=14/35 failed 2382 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1837, waitTime=2220, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20140 ms, replay 2382 ops.
14/07/02 17:08:43 INFO client.AsyncProcess: #18, table=usertable, attempt=14/35 failed 2495 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1837, waitTime=2220, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20191 ms, replay 2495 ops.
14/07/02 17:08:43 INFO client.AsyncProcess: #17, table=usertable, attempt=14/35 failed 2477 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1837, waitTime=2220, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20028 ms, replay 2477 ops.
14/07/02 17:08:43 INFO client.AsyncProcess: #10, table=usertable, attempt=14/35 failed 2511 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1837, waitTime=2220, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20130 ms, replay 2511 ops.
14/07/02 17:08:43 INFO client.AsyncProcess: #19, table=usertable, attempt=14/35 failed 2404 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1837, waitTime=2220, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20039 ms, replay 2404 ops.
14/07/02 17:08:43 INFO client.AsyncProcess: #20, table=usertable, attempt=14/35 failed 2458 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1837, waitTime=2220, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20054 ms, replay 2458 ops.
14/07/02 17:08:43 INFO client.AsyncProcess: #4, table=usertable, attempt=14/35 failed 2403 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1837, waitTime=2220, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20015 ms, replay 2403 ops.
14/07/02 17:08:43 INFO client.AsyncProcess: #2, table=usertable, attempt=14/35 failed 2443 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1837, waitTime=2220, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20151 ms, replay 2443 ops.
14/07/02 17:08:43 INFO client.AsyncProcess: #6, table=usertable, attempt=14/35 failed 2496 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1837, waitTime=2220, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20149 ms, replay 2496 ops.
14/07/02 17:08:43 INFO client.AsyncProcess: #12, table=usertable, attempt=14/35 failed 2383 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1837, waitTime=2220, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20030 ms, replay 2383 ops.
2014-07-02 17:08:51:838 150 sec: 1657097 operations; 24731.93 current ops/sec; [INSERT AverageLatency(us)=313.37] 
2014-07-02 17:09:01:839 160 sec: 1841145 operations; 18402.96 current ops/sec; [INSERT AverageLatency(us)=421.45] 
14/07/02 17:09:05 INFO client.AsyncProcess: #4, table=usertable, attempt=15/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014
14/07/02 17:09:05 INFO client.AsyncProcess: #12, table=usertable, attempt=15/35 failed 2383 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2247, waitTime=2007, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20170 ms, replay 2383 ops.
14/07/02 17:09:05 INFO client.AsyncProcess: #17, table=usertable, attempt=15/35 failed 2477 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2247, waitTime=2007, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20058 ms, replay 2477 ops.
14/07/02 17:09:05 INFO client.AsyncProcess: #1, table=usertable, attempt=15/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014
14/07/02 17:09:05 INFO client.AsyncProcess: #10, table=usertable, attempt=15/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014
14/07/02 17:09:05 INFO client.AsyncProcess: #13, table=usertable, attempt=15/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014
14/07/02 17:09:05 INFO client.AsyncProcess: #2, table=usertable, attempt=15/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014
14/07/02 17:09:05 INFO client.AsyncProcess: #19, table=usertable, attempt=15/35 failed 2404 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2251, waitTime=2289, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20059 ms, replay 2404 ops.
14/07/02 17:09:05 INFO client.AsyncProcess: #18, table=usertable, attempt=15/35 failed 2495 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2251, waitTime=2289, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20081 ms, replay 2495 ops.
14/07/02 17:09:05 INFO client.AsyncProcess: #20, table=usertable, attempt=15/35 failed 2458 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2251, waitTime=2289, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20168 ms, replay 2458 ops.
14/07/02 17:09:05 INFO client.AsyncProcess: #5, table=usertable, attempt=15/35 failed 2390 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2251, waitTime=2289, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20055 ms, replay 2390 ops.
14/07/02 17:09:05 INFO client.AsyncProcess: #6, table=usertable, attempt=15/35 failed 2496 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2251, waitTime=2289, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20048 ms, replay 2496 ops.
2014-07-02 17:09:11:840 170 sec: 2003514 operations; 16235.28 current ops/sec; [UPDATE AverageLatency(us)=3502152.5] [INSERT AverageLatency(us)=5218.06] [CLEANUP AverageLatency(us)=3503435.5] 
2014-07-02 17:09:21:875 180 sec: 2072653 operations; 6889.79 current ops/sec;  [INSERT AverageLatency(us)=1264.09]  
14/07/02 17:09:27 INFO client.AsyncProcess: #12, table=usertable, attempt=16/35 failed 2383 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2671, waitTime=2281, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20064 ms, replay 2383 ops.
14/07/02 17:09:27 INFO client.AsyncProcess: #17, table=usertable, attempt=16/35 failed 2477 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2671, waitTime=2281, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20169 ms, replay 2477 ops.
14/07/02 17:09:27 INFO client.AsyncProcess: #5, table=usertable, attempt=16/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014
14/07/02 17:09:27 INFO client.AsyncProcess: #20, table=usertable, attempt=16/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014
14/07/02 17:09:27 INFO client.AsyncProcess: #19, table=usertable, attempt=16/35 failed 2404 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2681, waitTime=2000, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20169 ms, replay 2404 ops.
14/07/02 17:09:27 INFO client.AsyncProcess: #6, table=usertable, attempt=16/35 failed 2496 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2679, waitTime=2009, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20145 ms, replay 2496 ops.
14/07/02 17:09:27 INFO client.AsyncProcess: #18, table=usertable, attempt=16/35 failed 2495 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2682, waitTime=2064, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20081 ms, replay 2495 ops.
2014-07-02 17:09:31:876 190 sec: 2379169 operations; 30651.6 current ops/sec;  [INSERT AverageLatency(us)=1671.3]  
2014-07-02 17:09:41:876 200 sec: 2449027 operations; 6985.1 current ops/sec;  [INSERT AverageLatency(us)=244.96]  
14/07/02 17:09:42 INFO client.AsyncProcess: #3, waiting for some tasks to finish. Expected max=0, tasksSent=272, tasksDone=271, currentTasksDone=271, retries=47 hasError=false, tableName=usertable
14/07/02 17:09:49 INFO client.AsyncProcess: #12, table=usertable, attempt=17/35 failed 2383 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3096, waitTime=3429, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20055 ms, replay 2383 ops.
14/07/02 17:09:49 INFO client.AsyncProcess: #17, table=usertable, attempt=17/35 failed 2477 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3096, waitTime=3429, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20076 ms, replay 2477 ops.
14/07/02 17:09:50 INFO client.AsyncProcess: #19, table=usertable, attempt=17/35 failed 2404 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3109, waitTime=2767, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20170 ms, replay 2404 ops.
14/07/02 17:09:50 INFO client.AsyncProcess: #6, table=usertable, attempt=17/35 failed 2496 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3109, waitTime=2767, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20099 ms, replay 2496 ops.
14/07/02 17:09:50 INFO client.AsyncProcess: #18, table=usertable, attempt=17/35 failed 2495 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3109, waitTime=2767, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20121 ms, replay 2495 ops.
2014-07-02 17:09:51:882 210 sec: 2449027 operations; 0 current ops/sec;    
14/07/02 17:09:53 INFO client.AsyncProcess: #3, waiting for some tasks to finish. Expected max=0, tasksSent=274, tasksDone=273, currentTasksDone=273, retries=49 hasError=false, tableName=usertable
2014-07-02 17:10:01:883 220 sec: 2631099 operations; 18205.38 current ops/sec; [UPDATE AverageLatency(us)=9555046] [INSERT AverageLatency(us)=752.05] [CLEANUP AverageLatency(us)=9555067.33] 
14/07/02 17:10:11 INFO client.AsyncProcess: #12, table=usertable, attempt=18/35 failed 2383 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3552, waitTime=2198, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20002 ms, replay 2383 ops.
14/07/02 17:10:11 INFO client.AsyncProcess: #17, table=usertable, attempt=18/35 failed 2477 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3555, waitTime=2025, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20079 ms, replay 2477 ops.
2014-07-02 17:10:11:907 230 sec: 2829817 operations; 19824.22 current ops/sec;  [INSERT AverageLatency(us)=319.27]  
14/07/02 17:10:12 INFO client.AsyncProcess: #19, table=usertable, attempt=18/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014
14/07/02 17:10:12 INFO client.AsyncProcess: #6, table=usertable, attempt=18/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014
14/07/02 17:10:12 INFO client.AsyncProcess: #18, table=usertable, attempt=18/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014
2014-07-02 17:10:21:908 240 sec: 3126792 operations; 29694.53 current ops/sec; [UPDATE AverageLatency(us)=2337325] [INSERT AverageLatency(us)=2452.78] [CLEANUP AverageLatency(us)=2337345] 
14/07/02 17:10:23 INFO client.AsyncProcess: #15, table=usertable, attempt=10/35 failed 2812 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3837, waitTime=2003, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:30 PDT 2014, retrying after 10006 ms, replay 2812 ops.
14/07/02 17:10:23 INFO client.AsyncProcess: #1, table=usertable, attempt=10/35 failed 2790 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3838, waitTime=2676, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014, retrying after 10008 ms, replay 2790 ops.
14/07/02 17:10:23 INFO client.AsyncProcess: #7, table=usertable, attempt=10/35 failed 2794 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3838, waitTime=2676, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014, retrying after 10052 ms, replay 2794 ops.
14/07/02 17:10:23 INFO client.AsyncProcess: #20, table=usertable, attempt=10/35 failed 2751 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3838, waitTime=2676, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014, retrying after 10061 ms, replay 2751 ops.
14/07/02 17:10:23 INFO client.AsyncProcess: #2, table=usertable, attempt=10/35 failed 2793 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3838, waitTime=2676, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014, retrying after 10025 ms, replay 2793 ops.
14/07/02 17:10:23 INFO client.AsyncProcess: #4, table=usertable, attempt=10/35 failed 2744 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3838, waitTime=2676, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014, retrying after 10097 ms, replay 2744 ops.
2014-07-02 17:10:31:908 250 sec: 3244706 operations; 11791.4 current ops/sec; [UPDATE AverageLatency(us)=1447372] [INSERT AverageLatency(us)=464.25] [CLEANUP AverageLatency(us)=1447395] 
14/07/02 17:10:34 INFO client.AsyncProcess: #17, table=usertable, attempt=19/35 failed 2477 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4016, waitTime=2446, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20139 ms, replay 2477 ops.
14/07/02 17:10:34 INFO client.AsyncProcess: #12, table=usertable, attempt=19/35 failed 2383 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4016, waitTime=2446, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014, retrying after 20051 ms, replay 2383 ops.
14/07/02 17:10:36 INFO client.AsyncProcess: #2, table=usertable, attempt=11/35 failed 2793 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4030, waitTime=3440, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014, retrying after 10066 ms, replay 2793 ops.
14/07/02 17:10:36 INFO client.AsyncProcess: #1, table=usertable, attempt=11/35 failed 2790 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4030, waitTime=3440, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014, retrying after 10006 ms, replay 2790 ops.
14/07/02 17:10:36 INFO client.AsyncProcess: #4, table=usertable, attempt=11/35 failed 2744 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4030, waitTime=3440, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014, retrying after 10060 ms, replay 2744 ops.
14/07/02 17:10:36 INFO client.AsyncProcess: #15, table=usertable, attempt=11/35 failed 2812 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4030, waitTime=3440, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:30 PDT 2014, retrying after 10026 ms, replay 2812 ops.
14/07/02 17:10:36 INFO client.AsyncProcess: #7, table=usertable, attempt=11/35 failed 2794 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4030, waitTime=3440, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014, retrying after 10053 ms, replay 2794 ops.
14/07/02 17:10:36 INFO client.AsyncProcess: #20, table=usertable, attempt=11/35 failed 2751 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4030, waitTime=3440, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014, retrying after 10025 ms, replay 2751 ops.
2014-07-02 17:10:41:911 260 sec: 3316776 operations; 7204.84 current ops/sec; [UPDATE AverageLatency(us)=867708] [INSERT AverageLatency(us)=739.81] [CLEANUP AverageLatency(us)=867728] 
14/07/02 17:10:48 INFO client.AsyncProcess: #15, table=usertable, attempt=12/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:30 PDT 2014
14/07/02 17:10:48 INFO client.AsyncProcess: #1, table=usertable, attempt=12/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014
14/07/02 17:10:48 INFO client.AsyncProcess: #20, table=usertable, attempt=12/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014
14/07/02 17:10:48 INFO client.AsyncProcess: #2, table=usertable, attempt=12/35 failed 2793 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4229, waitTime=2033, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014, retrying after 20055 ms, replay 2793 ops.
14/07/02 17:10:48 INFO client.AsyncProcess: #7, table=usertable, attempt=12/35 failed 2794 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4229, waitTime=2033, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014, retrying after 20167 ms, replay 2794 ops.
14/07/02 17:10:48 INFO client.AsyncProcess: #4, table=usertable, attempt=12/35 failed 2744 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4229, waitTime=2033, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014, retrying after 20023 ms, replay 2744 ops.
2014-07-02 17:10:51:912 270 sec: 3548576 operations; 23177.68 current ops/sec; [UPDATE AverageLatency(us)=38878042.5] [INSERT AverageLatency(us)=810.12] [CLEANUP AverageLatency(us)=38878064.5] 
14/07/02 17:10:55 INFO client.AsyncProcess: #12, table=usertable, attempt=20/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014
14/07/02 17:10:55 INFO client.AsyncProcess: #17, table=usertable, attempt=20/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:06:23 PDT 2014
2014-07-02 17:11:01:913 280 sec: 3767809 operations; 21921.11 current ops/sec;  [INSERT AverageLatency(us)=2683.27]  
14/07/02 17:11:10 INFO client.AsyncProcess: #4, table=usertable, attempt=13/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014
14/07/02 17:11:10 INFO client.AsyncProcess: #2, table=usertable, attempt=13/35 failed 2793 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4747, waitTime=2045, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014, retrying after 20135 ms, replay 2793 ops.
14/07/02 17:11:11 INFO client.AsyncProcess: #7, table=usertable, attempt=13/35 failed 2794 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4750, waitTime=2090, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014, retrying after 20078 ms, replay 2794 ops.
2014-07-02 17:11:11:961 290 sec: 3916589 operations; 14806.93 current ops/sec;  [INSERT AverageLatency(us)=1090.21]  
2014-07-02 17:11:21:961 300 sec: 4142667 operations; 22607.8 current ops/sec;  [INSERT AverageLatency(us)=406.38]  
2014-07-02 17:11:31:965 310 sec: 4417420 operations; 27464.31 current ops/sec; [UPDATE AverageLatency(us)=2032320.33] [INSERT AverageLatency(us)=185.43] [CLEANUP AverageLatency(us)=2032346] 
14/07/02 17:11:32 INFO client.AsyncProcess: #2, table=usertable, attempt=14/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014
14/07/02 17:11:32 INFO client.AsyncProcess: #7, table=usertable, attempt=14/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404345622776, tracking started Wed Jul 02 17:09:31 PDT 2014
2014-07-02 17:11:41:966 320 sec: 4779511 operations; 36205.48 current ops/sec; [UPDATE AverageLatency(us)=41387398] [INSERT AverageLatency(us)=459.95] [CLEANUP AverageLatency(us)=41387424] 
2014-07-02 17:11:50:629 329 sec: 5000000 operations; 25451.81 current ops/sec; [UPDATE AverageLatency(us)=935594.5] [INSERT AverageLatency(us)=105.09] [CLEANUP AverageLatency(us)=935627.25] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404346312
1404346312
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo 'random read...'
random read...
+ echo 'starting time: '
starting time: 
++ date +%s
+ echo 1404346322
1404346322
++ echo /home/hadoop/git/physical_design/caseresult/case5/random_read_1.dat
+ ./random_read 5000000 20 /home/hadoop/git/physical_design/caseresult/case5/random_read_1.dat
Loading workload...
Starting test.
2014-07-02 17:12:03:744 0 sec: 0 operations; 
14/07/02 17:12:05 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/02 17:12:05 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/02 17:12:05 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/02 17:12:05 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/02 17:12:05 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/02 17:12:05 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/02 17:12:05 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/02 17:12:05 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/02 17:12:05 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/02 17:12:05 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/02 17:12:05 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/02 17:12:05 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/02 17:12:05 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/02 17:12:05 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/02 17:12:05 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/02 17:12:05 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0x4ef888ea, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/02 17:12:05 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0x4ef888ea connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/02 17:12:05 INFO zookeeper.ClientCnxn: Opening socket connection to server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/02 17:12:05 INFO zookeeper.ClientCnxn: Socket connection established to sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, initiating session
14/07/02 17:12:05 INFO zookeeper.ClientCnxn: Session establishment complete on server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, sessionid = 0x46f985a17c000e, negotiated timeout = 90000
2014-07-02 17:12:13:746 10 sec: 67810 operations; 6745.25 current ops/sec; [READ AverageLatency(us)=2933.59] 
2014-07-02 17:12:23:748 20 sec: 211107 operations; 14326.83 current ops/sec; [READ AverageLatency(us)=1390.57] 
2014-07-02 17:12:33:749 30 sec: 363770 operations; 15266.3 current ops/sec; [READ AverageLatency(us)=1304.02] 
2014-07-02 17:12:43:749 40 sec: 518513 operations; 15472.75 current ops/sec; [READ AverageLatency(us)=1286.71] 
2014-07-02 17:12:53:750 50 sec: 672858 operations; 15432.96 current ops/sec; [READ AverageLatency(us)=1292.75] 
2014-07-02 17:13:03:750 60 sec: 820073 operations; 14721.5 current ops/sec; [READ AverageLatency(us)=1353.53] 
2014-07-02 17:13:13:751 70 sec: 994374 operations; 17428.36 current ops/sec; [READ AverageLatency(us)=1140.68] 
2014-07-02 17:13:23:751 80 sec: 1174389 operations; 18001.5 current ops/sec; [READ AverageLatency(us)=1107.95] 
2014-07-02 17:13:33:752 90 sec: 1362851 operations; 18844.32 current ops/sec; [READ AverageLatency(us)=1056.33] 
2014-07-02 17:13:43:753 100 sec: 1552482 operations; 18961.2 current ops/sec; [READ AverageLatency(us)=1049.73] 
2014-07-02 17:13:53:753 110 sec: 1739279 operations; 18679.7 current ops/sec; [READ AverageLatency(us)=1065.78] 
2014-07-02 17:14:03:754 120 sec: 1924665 operations; 18536.75 current ops/sec; [READ AverageLatency(us)=1073.65] 
2014-07-02 17:14:13:754 130 sec: 2110026 operations; 18536.1 current ops/sec; [READ AverageLatency(us)=1074.33] 
2014-07-02 17:14:23:755 140 sec: 2298446 operations; 18840.12 current ops/sec; [READ AverageLatency(us)=1056.53] 
2014-07-02 17:14:33:756 150 sec: 2484731 operations; 18628.5 current ops/sec; [READ AverageLatency(us)=1068.66] 
2014-07-02 17:14:43:756 160 sec: 2665862 operations; 18111.29 current ops/sec; [READ AverageLatency(us)=1099] 
2014-07-02 17:14:53:757 170 sec: 2853101 operations; 18722.03 current ops/sec; [READ AverageLatency(us)=1063.22] 
2014-07-02 17:15:03:758 180 sec: 3032976 operations; 17985.7 current ops/sec; [READ AverageLatency(us)=1106.92] 
2014-07-02 17:15:13:759 190 sec: 3198372 operations; 16537.95 current ops/sec; [READ AverageLatency(us)=1204.35] 
2014-07-02 17:15:23:760 200 sec: 3356982 operations; 15859.41 current ops/sec; [READ AverageLatency(us)=1255.9] 
2014-07-02 17:15:33:760 210 sec: 3519504 operations; 16252.2 current ops/sec; [READ AverageLatency(us)=1225.68] 
2014-07-02 17:15:43:761 220 sec: 3695751 operations; 17622.94 current ops/sec; [READ AverageLatency(us)=1129.37] 
2014-07-02 17:15:53:762 230 sec: 3880549 operations; 18477.95 current ops/sec; [READ AverageLatency(us)=1074.93] 
2014-07-02 17:16:03:763 240 sec: 4064722 operations; 18417.3 current ops/sec; [READ AverageLatency(us)=1083.4] 
2014-07-02 17:16:13:763 250 sec: 4253249 operations; 18850.81 current ops/sec; [READ AverageLatency(us)=1055.98] 
2014-07-02 17:16:23:764 260 sec: 4439800 operations; 18653.23 current ops/sec; [READ AverageLatency(us)=1067.25] 
2014-07-02 17:16:33:765 270 sec: 4620807 operations; 18098.89 current ops/sec; [READ AverageLatency(us)=1099.89] 
2014-07-02 17:16:43:766 280 sec: 4803356 operations; 18253.07 current ops/sec; [READ AverageLatency(us)=1088.91] 
2014-07-02 17:16:53:767 290 sec: 4985094 operations; 18171.98 current ops/sec; [READ AverageLatency(us)=1096.63] 
2014-07-02 17:16:54:971 291 sec: 5000000 operations; 12380.4 current ops/sec; [UPDATE AverageLatency(us)=12.85] [READ AverageLatency(us)=986.7] [CLEANUP AverageLatency(us)=281.45] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404346615
1404346615
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo workloada...
workloada...
+ echo 'start time: '
start time: 
++ date +%s
+ echo 1404346625
1404346625
++ echo /home/hadoop/git/physical_design/caseresult/case5/workloada_1.dat
+ ./workloada 5000000 20 /home/hadoop/git/physical_design/caseresult/case5/workloada_1.dat
Loading workload...
Starting test.
2014-07-02 17:17:05:972 0 sec: 0 operations; 
14/07/02 17:17:06 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/02 17:17:06 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/02 17:17:06 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/02 17:17:06 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/02 17:17:06 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/02 17:17:06 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/02 17:17:06 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/02 17:17:06 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/02 17:17:06 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/02 17:17:06 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/02 17:17:06 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/02 17:17:06 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/02 17:17:06 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/02 17:17:06 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/02 17:17:06 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/02 17:17:06 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0x6b355469, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/02 17:17:06 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0x6b355469 connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/02 17:17:06 INFO zookeeper.ClientCnxn: Opening socket connection to server slave1/9.1.143.59:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/02 17:17:06 INFO zookeeper.ClientCnxn: Socket connection established to slave1/9.1.143.59:2181, initiating session
14/07/02 17:17:06 INFO zookeeper.ClientCnxn: Session establishment complete on server slave1/9.1.143.59:2181, sessionid = 0x146f9859c3c000b, negotiated timeout = 90000
2014-07-02 17:17:15:976 10 sec: 35641 operations; 3562.32 current ops/sec; [INSERT AverageLatency(us)=910.39] [READ AverageLatency(us)=3193.38] 
2014-07-02 17:17:25:978 20 sec: 167452 operations; 13179.78 current ops/sec; [INSERT AverageLatency(us)=16.49] [READ AverageLatency(us)=1556.23] 
2014-07-02 17:17:35:979 30 sec: 310869 operations; 14340.27 current ops/sec; [INSERT AverageLatency(us)=13.76] [READ AverageLatency(us)=1528.86] 
2014-07-02 17:17:45:980 40 sec: 468654 operations; 15775.34 current ops/sec; [INSERT AverageLatency(us)=12.93] [READ AverageLatency(us)=1389.34] 
2014-07-02 17:17:55:980 50 sec: 623145 operations; 15449.1 current ops/sec; [INSERT AverageLatency(us)=13.9] [READ AverageLatency(us)=1421.02] 
2014-07-02 17:18:06:230 60 sec: 767112 operations; 14045.56 current ops/sec; [INSERT AverageLatency(us)=14.01] [READ AverageLatency(us)=1521.54] 
2014-07-02 17:18:16:231 70 sec: 909101 operations; 14198.9 current ops/sec; [INSERT AverageLatency(us)=68.23] [READ AverageLatency(us)=1597.09] 
2014-07-02 17:18:26:231 80 sec: 998762 operations; 8965.2 current ops/sec; [INSERT AverageLatency(us)=47.35] [READ AverageLatency(us)=2459.51] 
2014-07-02 17:18:36:232 90 sec: 1028815 operations; 3005 current ops/sec; [INSERT AverageLatency(us)=13.78] [READ AverageLatency(us)=1637.06] 
2014-07-02 17:18:46:233 100 sec: 1054547 operations; 2572.94 current ops/sec; [INSERT AverageLatency(us)=25.41] [READ AverageLatency(us)=15317.16] 
2014-07-02 17:18:56:234 110 sec: 1129994 operations; 7543.95 current ops/sec; [INSERT AverageLatency(us)=15.56] [READ AverageLatency(us)=2934.17] 
2014-07-02 17:19:06:235 120 sec: 1250802 operations; 12079.59 current ops/sec; [INSERT AverageLatency(us)=18.37] [READ AverageLatency(us)=1827.33] 
2014-07-02 17:19:16:235 130 sec: 1369237 operations; 11843.5 current ops/sec; [INSERT AverageLatency(us)=15.16] [READ AverageLatency(us)=1863.28] 
2014-07-02 17:19:26:236 140 sec: 1483652 operations; 11440.36 current ops/sec; [INSERT AverageLatency(us)=14.62] [READ AverageLatency(us)=1918.07] 
2014-07-02 17:19:36:237 150 sec: 1602239 operations; 11857.51 current ops/sec; [INSERT AverageLatency(us)=13.44] [READ AverageLatency(us)=1876.73] 
2014-07-02 17:19:46:237 160 sec: 1726232 operations; 12399.3 current ops/sec; [INSERT AverageLatency(us)=13.24] [READ AverageLatency(us)=1767.65] 
2014-07-02 17:19:56:238 170 sec: 1852269 operations; 12602.44 current ops/sec; [INSERT AverageLatency(us)=18.74] [READ AverageLatency(us)=1767.9] 
2014-07-02 17:20:06:239 180 sec: 1933579 operations; 8130.19 current ops/sec; [INSERT AverageLatency(us)=29.75] [READ AverageLatency(us)=2726.82] 
2014-07-02 17:20:16:239 190 sec: 2076094 operations; 14251.5 current ops/sec; [INSERT AverageLatency(us)=12.58] [READ AverageLatency(us)=1549.09] 
2014-07-02 17:20:26:240 200 sec: 2225994 operations; 14988.5 current ops/sec; [INSERT AverageLatency(us)=15.75] [READ AverageLatency(us)=1471.17] 
2014-07-02 17:20:36:241 210 sec: 2384007 operations; 15799.72 current ops/sec; [INSERT AverageLatency(us)=14.29] [READ AverageLatency(us)=1395.78] 
2014-07-02 17:20:46:242 220 sec: 2548525 operations; 16451.8 current ops/sec; [INSERT AverageLatency(us)=13.54] [READ AverageLatency(us)=1339.59] 
2014-07-02 17:20:56:242 230 sec: 2714237 operations; 16569.54 current ops/sec; [INSERT AverageLatency(us)=12.95] [READ AverageLatency(us)=1333.75] 
2014-07-02 17:21:06:243 240 sec: 2830472 operations; 11622.34 current ops/sec; [INSERT AverageLatency(us)=22.2] [READ AverageLatency(us)=1894.27] 
2014-07-02 17:21:16:244 250 sec: 2971836 operations; 14136.4 current ops/sec; [INSERT AverageLatency(us)=16] [READ AverageLatency(us)=1568.64] 
2014-07-02 17:21:26:244 260 sec: 3140797 operations; 16894.41 current ops/sec; [INSERT AverageLatency(us)=13.28] [READ AverageLatency(us)=1306.14] 
2014-07-02 17:21:36:245 270 sec: 3304876 operations; 16406.26 current ops/sec; [INSERT AverageLatency(us)=14.78] [READ AverageLatency(us)=1344.6] 
2014-07-02 17:21:46:246 280 sec: 3466361 operations; 16146.89 current ops/sec; [INSERT AverageLatency(us)=24.86] [READ AverageLatency(us)=1364.72] 
2014-07-02 17:21:56:247 290 sec: 3632901 operations; 16652.33 current ops/sec; [INSERT AverageLatency(us)=19.85] [READ AverageLatency(us)=1324.56] 
2014-07-02 17:22:06:247 300 sec: 3759236 operations; 12633.5 current ops/sec; [INSERT AverageLatency(us)=20.97] [READ AverageLatency(us)=1738.06] 
2014-07-02 17:22:16:248 310 sec: 3891708 operations; 13245.88 current ops/sec; [INSERT AverageLatency(us)=20.24] [READ AverageLatency(us)=1677.34] 
2014-07-02 17:22:26:249 320 sec: 4065587 operations; 17386.16 current ops/sec; [INSERT AverageLatency(us)=15.25] [READ AverageLatency(us)=1269.42] 
2014-07-02 17:22:36:250 330 sec: 4239184 operations; 17357.96 current ops/sec; [INSERT AverageLatency(us)=17.11] [READ AverageLatency(us)=1272.97] 
2014-07-02 17:22:46:250 340 sec: 4404285 operations; 16510.1 current ops/sec; [INSERT AverageLatency(us)=16.35] [READ AverageLatency(us)=1335.81] 
2014-07-02 17:22:56:251 350 sec: 4574440 operations; 17013.8 current ops/sec; [INSERT AverageLatency(us)=16.53] [READ AverageLatency(us)=1295.6] 
2014-07-02 17:23:06:252 360 sec: 4700170 operations; 12571.74 current ops/sec; [INSERT AverageLatency(us)=20.23] [READ AverageLatency(us)=1757.09] 
2014-07-02 17:23:16:253 370 sec: 4807277 operations; 10710.7 current ops/sec; [INSERT AverageLatency(us)=22.15] [READ AverageLatency(us)=2064.27] 
2014-07-02 17:23:26:253 380 sec: 4968878 operations; 16158.48 current ops/sec; [INSERT AverageLatency(us)=16.89] [READ AverageLatency(us)=1363.59] 
2014-07-02 17:23:30:328 384 sec: 5000000 operations; 7637.3 current ops/sec; [UPDATE AverageLatency(us)=151316.25] [INSERT AverageLatency(us)=17.01] [READ AverageLatency(us)=1727.6] [CLEANUP AverageLatency(us)=151699.9] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404347010
1404347010
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo workloadb...
workloadb...
+ echo 'start time: '
start time: 
++ date +%s
+ echo 1404347020
1404347020
++ echo /home/hadoop/git/physical_design/caseresult/case5/workloadb_1.dat
+ ./workloadb 5000000 20 /home/hadoop/git/physical_design/caseresult/case5/workloadb_1.dat
Loading workload...
Starting test.
2014-07-02 17:23:41:335 0 sec: 0 operations; 
14/07/02 17:23:41 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/02 17:23:41 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/02 17:23:41 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/02 17:23:41 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/02 17:23:41 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/02 17:23:41 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/02 17:23:41 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/02 17:23:41 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/02 17:23:41 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/02 17:23:41 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/02 17:23:41 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/02 17:23:41 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/02 17:23:41 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/02 17:23:41 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/02 17:23:41 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/02 17:23:41 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0x578ef2b6, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/02 17:23:41 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0x578ef2b6 connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/02 17:23:41 INFO zookeeper.ClientCnxn: Opening socket connection to server slave1/9.1.143.59:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/02 17:23:41 INFO zookeeper.ClientCnxn: Socket connection established to slave1/9.1.143.59:2181, initiating session
14/07/02 17:23:41 INFO zookeeper.ClientCnxn: Session establishment complete on server slave1/9.1.143.59:2181, sessionid = 0x146f9859c3c000c, negotiated timeout = 90000
2014-07-02 17:23:51:337 10 sec: 26803 operations; 2679.23 current ops/sec; [INSERT AverageLatency(us)=864.24] [READ AverageLatency(us)=4701.87] 
2014-07-02 17:24:01:339 20 sec: 187961 operations; 16112.58 current ops/sec; [INSERT AverageLatency(us)=14.79] [READ AverageLatency(us)=2070.8] 
2014-07-02 17:24:11:340 30 sec: 375743 operations; 18776.32 current ops/sec; [INSERT AverageLatency(us)=19.03] [READ AverageLatency(us)=2072.98] 
2014-07-02 17:24:21:341 40 sec: 560226 operations; 18448.3 current ops/sec; [INSERT AverageLatency(us)=18.55] [READ AverageLatency(us)=2139.1] 
2014-07-02 17:24:31:341 50 sec: 711704 operations; 15146.29 current ops/sec; [INSERT AverageLatency(us)=17.91] [READ AverageLatency(us)=2594.58] 
2014-07-02 17:24:41:342 60 sec: 837901 operations; 12618.44 current ops/sec; [INSERT AverageLatency(us)=19.05] [READ AverageLatency(us)=3155.95] 
2014-07-02 17:24:51:343 70 sec: 949599 operations; 11168.68 current ops/sec; [INSERT AverageLatency(us)=18.98] [READ AverageLatency(us)=3508.49] 
2014-07-02 17:25:01:344 80 sec: 1110165 operations; 16054.99 current ops/sec; [INSERT AverageLatency(us)=17.34] [READ AverageLatency(us)=2475.13] 
2014-07-02 17:25:11:345 90 sec: 1286087 operations; 17590.44 current ops/sec; [INSERT AverageLatency(us)=12.55] [READ AverageLatency(us)=2242.27] 
2014-07-02 17:25:21:346 100 sec: 1436075 operations; 14997.3 current ops/sec; [INSERT AverageLatency(us)=20.56] [READ AverageLatency(us)=2637.42] 
2014-07-02 17:25:31:347 110 sec: 1623719 operations; 18762.52 current ops/sec; [INSERT AverageLatency(us)=16.36] [READ AverageLatency(us)=2104.49] 
2014-07-02 17:25:41:348 120 sec: 1786935 operations; 16321.6 current ops/sec; [INSERT AverageLatency(us)=17.13] [READ AverageLatency(us)=2425.03] 
2014-07-02 17:25:51:349 130 sec: 1937001 operations; 15003.6 current ops/sec; [INSERT AverageLatency(us)=14.57] [READ AverageLatency(us)=2634.23] 
2014-07-02 17:26:01:350 140 sec: 2063010 operations; 12600.9 current ops/sec; [INSERT AverageLatency(us)=15.25] [READ AverageLatency(us)=3148.79] 
2014-07-02 17:26:11:350 150 sec: 2219201 operations; 15617.54 current ops/sec; [INSERT AverageLatency(us)=20.33] [READ AverageLatency(us)=2509.91] 
2014-07-02 17:26:21:351 160 sec: 2371644 operations; 15242.78 current ops/sec; [INSERT AverageLatency(us)=18.35] [READ AverageLatency(us)=2605.63] 
2014-07-02 17:26:31:351 170 sec: 2525716 operations; 15407.2 current ops/sec; [INSERT AverageLatency(us)=19.63] [READ AverageLatency(us)=2562.88] 
2014-07-02 17:26:41:352 180 sec: 2673926 operations; 14819.52 current ops/sec; [INSERT AverageLatency(us)=17.51] [READ AverageLatency(us)=2677.36] 
2014-07-02 17:26:51:353 190 sec: 2821347 operations; 14740.63 current ops/sec; [INSERT AverageLatency(us)=19.7] [READ AverageLatency(us)=2679.59] 
2014-07-02 17:27:01:353 200 sec: 2986773 operations; 16542.6 current ops/sec; [INSERT AverageLatency(us)=16.26] [READ AverageLatency(us)=2373.65] 
2014-07-02 17:27:11:354 210 sec: 3096920 operations; 11013.6 current ops/sec; [INSERT AverageLatency(us)=20.15] [READ AverageLatency(us)=3611.89] 
2014-07-02 17:27:21:355 220 sec: 3210934 operations; 11400.26 current ops/sec; [INSERT AverageLatency(us)=19.28] [READ AverageLatency(us)=3474.34] 
2014-07-02 17:27:31:355 230 sec: 3359087 operations; 14815.3 current ops/sec; [INSERT AverageLatency(us)=16.28] [READ AverageLatency(us)=2649.62] 
2014-07-02 17:27:41:356 240 sec: 3483728 operations; 12462.85 current ops/sec; [INSERT AverageLatency(us)=16.06] [READ AverageLatency(us)=3207.96] 
2014-07-02 17:27:51:437 250 sec: 3629520 operations; 14462.06 current ops/sec; [INSERT AverageLatency(us)=15.43] [READ AverageLatency(us)=2719.94] 
2014-07-02 17:28:01:437 260 sec: 3778048 operations; 14852.8 current ops/sec; [INSERT AverageLatency(us)=15.74] [READ AverageLatency(us)=2682.5] 
2014-07-02 17:28:11:439 270 sec: 3936665 operations; 15860.11 current ops/sec; [INSERT AverageLatency(us)=14.74] [READ AverageLatency(us)=2479.3] 
2014-07-02 17:28:21:439 280 sec: 4092246 operations; 15556.54 current ops/sec; [INSERT AverageLatency(us)=16.68] [READ AverageLatency(us)=2531.09] 
2014-07-02 17:28:31:440 290 sec: 4202684 operations; 11042.7 current ops/sec; [INSERT AverageLatency(us)=15.83] [READ AverageLatency(us)=3610.99] 
2014-07-02 17:28:41:441 300 sec: 4361895 operations; 15919.51 current ops/sec; [INSERT AverageLatency(us)=16.13] [READ AverageLatency(us)=2480.18] 
2014-07-02 17:28:51:442 310 sec: 4535038 operations; 17312.57 current ops/sec; [INSERT AverageLatency(us)=18.83] [READ AverageLatency(us)=2273.24] 
2014-07-02 17:29:01:443 320 sec: 4716561 operations; 18150.48 current ops/sec; [INSERT AverageLatency(us)=15.7] [READ AverageLatency(us)=2166.45] 
2014-07-02 17:29:11:444 330 sec: 4899668 operations; 18308.87 current ops/sec; [INSERT AverageLatency(us)=14.85] [READ AverageLatency(us)=2163.08] 
2014-07-02 17:29:18:792 337 sec: 5000000 operations; 13654.33 current ops/sec; [UPDATE AverageLatency(us)=247526.45] [INSERT AverageLatency(us)=16.7] [READ AverageLatency(us)=2162.2] [CLEANUP AverageLatency(us)=247925.85] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404347359
1404347359
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo workloadc...
workloadc...
+ echo 'start time: '
start time: 
++ date +%s
+ echo 1404347369
1404347369
++ echo /home/hadoop/git/physical_design/caseresult/case5/workloadc_1.dat
+ ./workloadc 5000000 20 /home/hadoop/git/physical_design/caseresult/case5/workloadc_1.dat
Loading workload...
Starting test.
2014-07-02 17:29:30:472 0 sec: 0 operations; 
14/07/02 17:29:30 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/02 17:29:30 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/02 17:29:30 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/02 17:29:30 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/02 17:29:30 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/02 17:29:30 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/02 17:29:30 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/02 17:29:30 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/02 17:29:30 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/02 17:29:30 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/02 17:29:30 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/02 17:29:30 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/02 17:29:30 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/02 17:29:30 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/02 17:29:30 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/02 17:29:30 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0xc2c915f, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/02 17:29:30 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0xc2c915f connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/02 17:29:30 INFO zookeeper.ClientCnxn: Opening socket connection to server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/02 17:29:31 INFO zookeeper.ClientCnxn: Socket connection established to sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, initiating session
14/07/02 17:29:31 INFO zookeeper.ClientCnxn: Session establishment complete on server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, sessionid = 0x46f985a17c000f, negotiated timeout = 90000
2014-07-02 17:29:40:475 10 sec: 205229 operations; 20477.85 current ops/sec; [INSERT AverageLatency(us)=146.77] [READ AverageLatency(us)=6867.19] 
2014-07-02 17:29:50:478 20 sec: 424138 operations; 21884.33 current ops/sec; [INSERT AverageLatency(us)=27.98] [READ AverageLatency(us)=8637.08] 
2014-07-02 17:30:00:479 30 sec: 706335 operations; 28216.88 current ops/sec; [INSERT AverageLatency(us)=12.8] [READ AverageLatency(us)=6935.77] 
2014-07-02 17:30:10:479 40 sec: 944187 operations; 23785.2 current ops/sec; [INSERT AverageLatency(us)=16.44] [READ AverageLatency(us)=8180.85] 
2014-07-02 17:30:20:480 50 sec: 1172091 operations; 22788.12 current ops/sec; [INSERT AverageLatency(us)=20.31] [READ AverageLatency(us)=8516.33] 
2014-07-02 17:30:30:496 60 sec: 1406448 operations; 23398.26 current ops/sec; [INSERT AverageLatency(us)=16.31] [READ AverageLatency(us)=8288.35] 
2014-07-02 17:30:40:497 70 sec: 1629750 operations; 22327.97 current ops/sec; [INSERT AverageLatency(us)=16.65] [READ AverageLatency(us)=8610.21] 
2014-07-02 17:30:50:497 80 sec: 1825698 operations; 19594.8 current ops/sec; [INSERT AverageLatency(us)=18.28] [READ AverageLatency(us)=10093.22] 
2014-07-02 17:31:00:498 90 sec: 2007385 operations; 18166.88 current ops/sec; [INSERT AverageLatency(us)=15.38] [READ AverageLatency(us)=10832.28] 
2014-07-02 17:31:10:499 100 sec: 2193688 operations; 18628.44 current ops/sec; [INSERT AverageLatency(us)=23.62] [READ AverageLatency(us)=10420] 
2014-07-02 17:31:20:500 110 sec: 2383796 operations; 19010.8 current ops/sec; [INSERT AverageLatency(us)=13.52] [READ AverageLatency(us)=10139.94] 
2014-07-02 17:31:30:500 120 sec: 2561201 operations; 17738.73 current ops/sec; [INSERT AverageLatency(us)=14.87] [READ AverageLatency(us)=11291.31] 
2014-07-02 17:31:40:501 130 sec: 2741984 operations; 18076.49 current ops/sec; [INSERT AverageLatency(us)=15.98] [READ AverageLatency(us)=10756.1] 
2014-07-02 17:31:50:502 140 sec: 2917206 operations; 17522.2 current ops/sec; [INSERT AverageLatency(us)=12.85] [READ AverageLatency(us)=11455.22] 
2014-07-02 17:32:00:502 150 sec: 3122379 operations; 20515.25 current ops/sec; [INSERT AverageLatency(us)=16.05] [READ AverageLatency(us)=9512.88] 
2014-07-02 17:32:10:503 160 sec: 3306021 operations; 18362.36 current ops/sec; [INSERT AverageLatency(us)=29.01] [READ AverageLatency(us)=10450.41] 
2014-07-02 17:32:20:507 170 sec: 3481460 operations; 17536.89 current ops/sec; [INSERT AverageLatency(us)=21.23] [READ AverageLatency(us)=11084.73] 
2014-07-02 17:32:30:508 180 sec: 3531547 operations; 5008.2 current ops/sec; [INSERT AverageLatency(us)=13.83] [READ AverageLatency(us)=39302.94] 
2014-07-02 17:32:40:509 190 sec: 3631448 operations; 9989.1 current ops/sec; [INSERT AverageLatency(us)=23.89] [READ AverageLatency(us)=19337.17] 
2014-07-02 17:32:50:510 200 sec: 3741329 operations; 10987 current ops/sec; [INSERT AverageLatency(us)=17.03] [READ AverageLatency(us)=16706.72] 
2014-07-02 17:33:00:511 210 sec: 3843654 operations; 10231.48 current ops/sec; [INSERT AverageLatency(us)=13.39] [READ AverageLatency(us)=21311.43] 
2014-07-02 17:33:10:514 220 sec: 3980470 operations; 13677.5 current ops/sec; [INSERT AverageLatency(us)=15.75] [READ AverageLatency(us)=14339.02] 
2014-07-02 17:33:20:517 230 sec: 4118028 operations; 13753.05 current ops/sec; [INSERT AverageLatency(us)=44.98] [READ AverageLatency(us)=13937.6] 
2014-07-02 17:33:30:528 240 sec: 4218941 operations; 10079.2 current ops/sec; [INSERT AverageLatency(us)=30.11] [READ AverageLatency(us)=19698.08] 
2014-07-02 17:33:40:530 250 sec: 4330123 operations; 11115.98 current ops/sec; [INSERT AverageLatency(us)=25.24] [READ AverageLatency(us)=17710.8] 
2014-07-02 17:33:50:531 260 sec: 4451099 operations; 12096.39 current ops/sec; [INSERT AverageLatency(us)=19.55] [READ AverageLatency(us)=16309.61] 
2014-07-02 17:34:00:544 270 sec: 4556133 operations; 10489.76 current ops/sec; [INSERT AverageLatency(us)=16.1] [READ AverageLatency(us)=18875.48] 
2014-07-02 17:34:10:544 280 sec: 4679904 operations; 12377.1 current ops/sec; [INSERT AverageLatency(us)=29.03] [READ AverageLatency(us)=15824.65] 
2014-07-02 17:34:20:545 290 sec: 4802314 operations; 12239.78 current ops/sec; [INSERT AverageLatency(us)=19.69] [READ AverageLatency(us)=16429.27] 
2014-07-02 17:34:30:549 300 sec: 4941873 operations; 13950.32 current ops/sec; [UPDATE AverageLatency(us)=579289.5] [INSERT AverageLatency(us)=16.91] [READ AverageLatency(us)=13701.74] [CLEANUP AverageLatency(us)=581135.5] 
2014-07-02 17:34:40:431 309 sec: 5000000 operations; 5882.11 current ops/sec; [UPDATE AverageLatency(us)=615877.72] [INSERT AverageLatency(us)=17.51] [READ AverageLatency(us)=14424.31] [CLEANUP AverageLatency(us)=615899.61] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404347681
1404347681
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ let counter=1+1
+ read line
+ set +x
